samples/go/photo-dedup/job/hashjob.go

```diff
@@ -33,51 +33,50 @@ func HashPhotosJob(db datas.Database, photoSets []types.Value, outDS datas.Datas
 
 // hashPhotos adds a dhash to each photo in photoSets and delivers them on the returned channel
 func hashPhotos(db datas.Database, photoSets []types.Value) <-chan types.Struct {
-	photosToHashCh := make(chan model.Photo)
-	hashedPhotosCh := make(chan types.Struct)
+	numWorkers := runtime.NumCPU() * 4
```

Comment:
It does! This is the result for gofmt. The idea is to remove spaces around hi precedence operators when lower precedence operators are also present.